pom.xml

```diff
@@ -492,6 +492,10 @@
           <artifactId>maven-failsafe-plugin</artifactId>
           <version>3.0.0-M3</version>
         </plugin>
+        <plugin>
```

Comment:
Not sure why, but without adding definition here, use of `mvn wrapper:wrapper` fails.